BrainPortal/config/initializers/added_core_extensions/object.rb

```diff
@@ -26,6 +26,25 @@
 class Object #:nodoc:
 
   include CBRAINExtensions::ObjectExtensions::DeepClone
-  
+
+  # This method returns the value of the class constant
```

Comment:
Put these two methods in a separate module file that will be included here, just like for DeepClone above.